src/services/aws/pricing/Pricing.ts

```diff
@@ -0,0 +1,17 @@
+import { PricingClient, PricingClientConfig } from "@aws-sdk/client-pricing";
+import { Product } from "../../../interfaces/product";
+import { getProducts } from "./function/GetProducts";
+import { writeProductsToFile } from "./function/WriteProductsToFile";
```

Comment:
Importing this function, but where is it used?

Reply:
Will remove